docs/sources/clients/promtail/scraping.md

```diff
@@ -64,6 +64,37 @@ There are different types of labels present in Promtail:
   uniqueness of the streams. It is set to the absolute path of the file the line
   was read from.
 
+### Example of File Discovery  
+To scrape a set of log files defined manually on a machine, one can use `static_configs` or `file_sd_cnfigs`. Using static_configs, one need to reload Promtail to apply modifications. Using `file_sd_configs` that reload is not needed. Promtail reloads discovery files when they are updated.  
```

Comment:
There's a misspelling here  `file_sd_cnfigs` -> `file_sd_configs`. I think also `one need to` should be `one needs to` or `one must`.